src/app/app-provider-admin/provider-admin/activities/employee-master-bulk-upload/employee-master-bulk-upload.component.html

```
@@ -0,0 +1,81 @@
+        <mat-label for="upload-file" *ngIf="file === undefined"
+          >Upload File(Size :0/{{ maxFileSize }}MB), Supported File
+          Formats(.xls, .xlsx, .xlsm, .xlsb)</mat-label
+        >
+        <input
+          class="m-t-10"
+          style="margin-left: 60px"
+          id="upload-file"
+          type="file"
+          accept=".xls, .xlsx, .xlsm, .xlsb"
+          size="100"
+          id="upload-file"
```

HTMLHint (1.5.0)

[error] Duplicate of attribute name [ id ] was found.

(attr-no-duplication)